Dmitriy_Vysotskyy2.0/Steps/CreateUserSteps.cs

```diff
@@ -0,0 +1,56 @@
+using Dmitriy_Vysotskyy2._0.Builders;
+using Dmitriy_Vysotskyy2._0.Common;
+using Dmitriy_Vysotskyy2._0.Models;
+using FluentAssertions;
+using TechTalk.SpecFlow;
+
+namespace Dmitriy_Vysotskyy2._0.Steps;
+
+[Binding]
+public class CreateUserSteps : FeatureHelper
+{
+    private TestUserModel _user;
+    private string _alertText;
+    
+    [Given(@"a user has come to clicked sign up")]
+    public void GivenAUserHasComeToClickedSignUp()
+    {
+        
+    }
+
+    [Given(@"a user doesn't have valid data")]
+    public void GivenAUserDoesntHaveValidData()
+    {
+        _user = new UserModelBuilder()
+            .SetLogin("charles")
+            .SetPassword("charles")
+            .Build();
+    }
+
+    [When(@"user inserts that data")]
+    public void WhenUserInsertsThatData()
+    {
+        _alertText = _homePage.SignUp(_user.Login,_user.Password);
+    }
+    
+
+    [Given(@"a user has valid data")]
```

Comment:
This description a bit confusing, user can't has valid data. Let's remove first empty step and rename this and step for random to something like:

"Valid data for login has been prepared" / "Random data for login has been prepared"